src/pe_reports/data/db_query.py

```diff
@@ -21,6 +22,16 @@
 CONN_PARAMS_DIC = config()
 
 
+def sanitize_string(string):
+    """Remove special characters from string."""
+    return re.sub(r"[^a-zA-Z0-9]", "", string)
+
+
+def sanitize_uid(string):
+    """Remove special characters from uids."""
+    return re.sub(r"[^a-zA-Z0-9\-]", "", string)
+
+
```

Comment:
I'll approve this, but shouldn't these functions be defined in one central place instead of being repeated in several files?

Reply:
Commit f2aedefe6dc7f96600c0eace469b28792be1de81 Imported the reused function, renamed other one as it had different functionality

Reply:
@jasonodoom, please allow the original reporter to resolve conversations.  That is the person who is in the best position to know whether or not the issue was satisfactorily resolved.

Reply:
Partially resolved.  Please see [here](https://github.com/cisagov/pe-reports/pull/597#discussion_r1255962405).

Reply:
Resolved in commit b04a2ff821024367373374139f3871c56a496b99